azure_functions_worker/dispatcher.py

```diff
@@ -678,21 +680,30 @@ def index_functions(self, function_path: str):
                     len(indexed_functions))
 
         if indexed_functions:
-            fx_metadata_results = loader.process_indexed_function(
-                self._functions,
-                indexed_functions)
+            fx_metadata_results, fx_bindings_logs = (
+                loader.process_indexed_function(
+                    self._functions,
+                    indexed_functions))
 
             indexed_function_logs: List[str] = []
+            bindings_info = []
             for func in indexed_functions:
+                fx_bindings = fx_bindings_logs.get(func)
```

Comment:
add log - it is only for log and not the actual bindigns.

Reply:
I meant suffix - log

Reply:
Renamed vars to match with others. (indexed_function_logs & indexed_function_bindings_logs), (func & func_binding_logs)